infra/gcp/terraform/k8s-infra-elections/variables.tf

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+variable "region" {
+  type        = string
+  default     = "us-central1"
+  description = "The default region for resources in the project"
+}
```

Comment:
Out of curiosity why variables vs. locals for all these?

Reply:
Input validation is only available for variables : https://www.terraform.io/docs/language/values/variables.html#custom-validation-rules. Also I developed the habit to use locals for programmatically-derived values based on variables.